scripts/createFromTemplate.ps1

```diff
@@ -141,6 +141,7 @@ Copy-Item "$templateFolder/../scripts/createDockerComposeProduction.ps1" "$locat
 Copy-Item "$templateFolder/../scripts/torizonPackages.ps1" "$location/.conf"
 Copy-Item "$templateFolder/../scripts/tasks.ps1" "$location/.vscode"
 Copy-Item "$templateFolder/../scripts/bash/tcb-env-setup.sh" "$location/.conf"
+Copy-Item "$templateFolder/../scripts/bash/bashrc.tcb" "$location/.conf/.bashrc.tcb"
```

Comment:
Also, this can be removed as the script will be inside the `tcb` folder

Reply:
Done and tested.